From db138b9ba12a0de5d6140832c0679c2418e3e7e0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Michel=20D=C3=A4nzer?= <michel.daenzer@amd.com>
Date: Thu, 21 Jan 2016 18:08:49 +0900
Subject: [PATCH 010/117] radeon: Pass radeon_bo_open flags to the
 DRM_RADEON_GEM_CREATE ioctl

Not doing so makes it impossible for radeon_bo_open callers to set any
RADEON_GEM_* flags for the newly created BO.

Reviewed-by: Alex Deucher <alexander.deucher@amd.com>
---
 radeon/radeon_bo_gem.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/radeon/radeon_bo_gem.c b/radeon/radeon_bo_gem.c
index c9fe19f..fbd453d 100644
--- a/radeon/radeon_bo_gem.c
+++ b/radeon/radeon_bo_gem.c
@@ -103,7 +103,7 @@ static struct radeon_bo *bo_open(struct radeon_bo_manager *bom,
         args.size = size;
         args.alignment = alignment;
         args.initial_domain = bo->base.domains;
-        args.flags = 0;
+        args.flags = flags;
         args.handle = 0;
         r = drmCommandWriteRead(bom->fd, DRM_RADEON_GEM_CREATE,
                                 &args, sizeof(args));
-- 
2.7.4

